library/std/src/os/unix/process.rs

```diff
@@ -197,6 +198,16 @@ pub trait CommandExt: Sealed {
     /// ```
     #[stable(feature = "process_set_process_group", since = "1.64.0")]
     fn process_group(&mut self, pgroup: i32) -> &mut process::Command;
+
+    /// Set the root of the child process. This calls `chroot` in the child process before executing
+    /// the command.
+    ///
+    /// This happens before changing to the directory specified with `Command::current_dir`, and
+    /// that directory will be relative to the new root. If no directory has been specified with
+    /// `Command::current_dir`, this will set the directory to `/`, to avoid leaving the current
+    /// directory outside the chroot.
```

Comment:
Usually, Rust is a thin wrapper around operating system functionality. The idea is that the user can implement `chroot` themselves if they want a `chroot` with a working directory outside of it?

Reply:
@tbu- Yes. The use case for doing that *intentionally* is generally exploits. But it can easily happen *accidentally*.

Reply:
Maybe you can point out more explicitly that this is a difference to the operating system feature `chroot`.

Reply:
Done.